src/modules/core/components/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useRef } from 'react';
+import { MessageDescriptor, useIntl } from 'react-intl';
+import { nanoid } from 'nanoid';
+import { useField } from 'formik';
+
+import { getMainClasses } from '~utils/css';
+import Icon from '~core/Icon';
+import { SimpleMessageValues } from '~types/index';
+
+import styles from './RadioButton.css';
+
+export interface Appearance {
+  theme?: 'primary' | 'danger';
+  direction?: 'horizontal' | 'vertical';
+}
+
+export interface RadioButtonTypes {
```

Comment:
why `Types` & not `Props`? I thought props was a default.

Reply:
Default of what? This is just name of interface